openitp-good-packaging-for-LAFS_sources-of-nondeterminism.rst

```diff
@@ -186,6 +186,8 @@ This approach can work only if, for each platform, the ➾ operation
 performed by the distributor results in a bytewise-identical binary
 to the ➾ operation performed by the security auditor.
 
+.. This is not strictly true, for example, a "fuzzy fingerprinting" algorithm could produce an identifier using only a subset of the build output bytes, for example by excluding known timestamps fields from a hash calculation.  That seems like a messy route and the one presented here is more sensical, so this is somewhat pedantic.
+
```

Comment:
Now that I think about it, it would be easier to zero out the nondeterministic fields, even in the case where tools cannot be changed. So I do not intend to add anything here.